tests/api/v2/datadog/security_monitoring_test.go

```diff
@@ -189,6 +189,11 @@ func TestSearchSecurityMonitoringSignals(t *testing.T) {
 	client := Client(ctx)
 	api := client.SecurityMonitoringApi
 
+	if tests.GetRecording() != tests.ModeReplaying {
```

Comment:
Hmm, so why not do `if tests.GetRecording() == tests.ModeIgnore` to only skip when running API tests and not recording?